assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -134,20 +137,31 @@ function InnerElement({
     width: width,
     height: height,
     alt: alt,
-    'aria-label': alt,
   };
-  const cloneProps = {
+
+  if (src && !src.startsWith(siteUrl)) {
+    commonProps.crossOrigin = 'anonymous';
+  }
+
+  const commonImageProps = {
     ...commonProps,
+    onLoad: makeMediaVisible,
```

Comment:
Why was `onLoad` moved from `imageProps` to the common props here? 🤔

Reply:
> I have also improved poster images, by making sure, loading=lazy, alt and onload are loaded on poster images as well.

Reply:
This just makes the image props between poster image and image the same.

Reply:
But I don't think the poster needs an `onLoad` callback?

Reply:
The onload just fades in the image. Poster image are image tags like an image, they should fail in the same way a image does.

Reply:
Also It already has a onload- https://github.com/google/web-stories-wp/pull/7086/files#diff-156fd9c14d28faad8ca9b94fd115f288b4a1511b74ac5b3c2bdd5db277f0ab91L193 This isnt a change.